addon/chrome.manifest

```diff
@@ -1 +1,2 @@
 content screenshots-skin chrome/skin/
+content screenshots chrome/content/
```

Comment:
actually, you want `content screenshots chrome/`.

Otherwise, you'll need the chrome URL to be `screenshots/content/content/foo.jsm`